src/Compilers/Core/Portable/MemberDescriptor.cs

```diff
@@ -72,7 +72,7 @@ public string DeclaringTypeMetadataName
 
         public MemberDescriptor(
             MemberFlags Flags,
-            byte DeclaringTypeId,
+            short DeclaringTypeId,
             string Name,
             ImmutableArray<byte> Signature,
             ushort Arity = 0)
```

Comment:
Not part of this change but consider changing parameter names to have lowercase first character.